src/components/CharacterCard.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="character-card">
+    <div class="character-card__image-wrapper">
+      <img class="character-card__image" :src="image" :alt="name" />
+    </div>
+    <div class="character-card__desc">
```

Comment:
from semantic perspective it should be probably 
ul
  li
    h1
  li
  li
    p
  li...